### ISSUE TITLE
plasma cutter fires in random directions

### ISSUE THREAD

**Issue by Serxule** [tgstation/tgstation#64257]
## Reproduction: https://www.youtube.com/watch?v=0fYhtqOERNQ
fire plasma cutter normally for a while, then suddenly it fires in random directions, maybe related to not having mesons on and clicking on the black? that doesnt really make sense but its the only thing i can think of.

**Comment by TemporalOroboros** [comment]
There's a fix for this that's on master, but it hasn't made it to the servers yet.

**Comment by TemporalOroboros** [comment]
Basically, when you click on an object you're supposed to shoot at the exact pixel you clicked on. Usually the pixel on the thing you clicked on is measured from the bottom left corner of the thing you clicked on. When you click on the black the game tries to find the turf you clicked on even if you can't see it, but it's using the pixel as measured from the bottom left corner of your screen instead of the bottom left corner of the turf.

TLDR: The game thinks you clicked about twice as far from the bottom left corner of the screen as you actually did when you click somewhere you can't see. There's a fix up and the next time the admins update the server it should be fixed.
